lib/trace_events.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const {
+  startTracing,
+  stopTracing,
+  getTracingCategories
+} = process.binding('trace_events');
+
+const { trace_events_enabled } = process.binding('config');
```

Comment:
Nit: If we can rename `trace_events_enabled` to something which is not in snake case here, it would be better.